File: src/question/month07/day0709/ms_0709.cs
using System;

namespace algorithm_09
{
    class ms_0709
    {
        static void Main(string[] args)
        { 
            char cnd;
            
            do {
                Console.Write("Base = ");
                int width = Convert.ToInt32(Console.ReadLine());
                
                Console.Write("Height = ");
                int height = Convert.ToInt32(Console.ReadLine());

                Console.WriteLine($"Triangle width = {width*height/2.0:.0}");
                
                Console.Write("Continue? ");
                cnd = Convert.ToChar(Console.ReadLine());
            } while(cnd == 'Y' || cnd == 'y');
        }
    }
}

File: year2020/month07/day0709/ms_0709.cpp
#include <iostream>

using namespace std;

int main() {
	int base, height;
	char cnd;

	do {
		cout << "Base = ";
		cin >> base;

		cout << "Height = ";
		cin >> height;

		printf("Triangle width = %.1lf\n", (base * height / 2.0));
		
		cout << "Continue? ";
		cin >> cnd;
	} while (cnd == 'Y' || cnd == 'y');

	return 0;
}